students/Osiddiquee/lesson04/trigram.py

```diff
@@ -0,0 +1,34 @@
+import random
+
+#reading in a text file
+inputfile = open('sherlock_small.txt')
```

Comment:
We wanna use a "with" context manager statement to open files:
```
with open('sherlock_small.txt') as file:
    novel = file.read()
```
This closes the file once you're de-indented so you don't need file.close().